api/controllers/CampaignBotController.js

```diff
@@ -225,13 +227,36 @@ class CampaignBotController {
   }
 
   /**
-   * Returns whether incomingMessage should begin a Reportback conversation.
-   * @param {string} incomingMessage
+   * Returns whether incoming request is an authorized cache clear command.
+   * @param {object} req
    * @return {bool}
    */
-  isCommandReportback(incomingMessage) {
-    const firstWord = helpers.getFirstWord(incomingMessage);
-    return ( firstWord && firstWord.toUpperCase() === CMD_REPORTBACK );
+  isCommandClearCache(req) {
+    const cmdClear =  process.env.GAMBIT_CMD_CLEAR_CACHE;
+
+    if (!cmdClear) return false;
+
+    if (req.user.role !== 'staff' && req.user.role !== 'admin') {
+      logger.warn(`${this.loggerPrefix(req)} unauthorized clear cache`);
+
+      return false;
+    }
+
+    const input = helpers.getFirstWord(req.incoming_message);
+
+    return ( input && input.toUpperCase() === cmdClear.toUpperCase() );
```

Comment:
Extra parentheses. Move the expression to own line and assign to a var so it's easier to debug.